deploy_website.sh

```diff
@@ -56,7 +56,7 @@ git checkout $SHA
 (cd kotlin && ./gradlew dokka)
 
 # Generate the Swift API docs.
-(cd swift/Samples/SampleApp && $SWIFT_DOCS_SCRIPT $SWIFT_API_DIR)
+($SWIFT_DOCS_SCRIPT $SWIFT_API_DIR)
```

Comment:
Doesn't the swift script need to be updated to use different relative paths?

Reply:
@amorde changed this, `Development.podspec` is now at the root.

Reply:
Oh cool. So this has just been flat out wrong for a while?

Reply:
Since https://github.com/square/workflow/pull/964

Reply:
This doesn't need the parentheses anymore – that was just there to scope the `cd`.
```suggestion
$SWIFT_DOCS_SCRIPT $SWIFT_API_DIR
```